.env.local.example

```diff
@@ -99,3 +99,8 @@ RADVIS_WFS_PASSWORD=secret
 # SVZBW DATEX2 Download URL
 ROADWORKS_SVZBW_DATEX2_DOWNLOAD_URL=https://url
 
+# Webcam
+WEBCAM_FTP_SERVER=server
+WEBCAM_FTP_USER=user
+WEBCAM_FTP_PASSWORD=password
+WEBCAM_BASIC_AUTH=secret
```

Comment:
Please add an explanation what this environment variable does and what its format looks like.

Reply:
Done